webapp/src/main/resources/build/template.soy.js

```diff
@@ -0,0 +1,18 @@
+// This file was automatically generated from template.soy.
+// Please don't edit this file by hand.
+
+if (typeof validator == 'undefined') { var validator = {}; }
+if (typeof validator.templates == 'undefined') { validator.templates = {}; }
+
+
+validator.templates.unknownColumnNotice = function(opt_data, opt_ignored) {
+  var output = '<p class="warning">Warning - Unknown Column(s) found!</p><p><b>' + soy.$$escapeHtml(opt_data.numNotices) + '</b> unknown column(s) found in:</p>';
+  var noticeList6 = opt_data.notices;
+  var noticeListLen6 = noticeList6.length;
+  for (var noticeIndex6 = 0; noticeIndex6 < noticeListLen6; noticeIndex6++) {
+    var noticeData6 = noticeList6[noticeIndex6];
+    output += '<p>Filename: <i>' + soy.$$escapeHtml(noticeData6.filename) + '</i></p><p>Column name: <i>' + soy.$$escapeHtml(noticeData6.fieldName) + '</i> at position <i>' + soy.$$escapeHtml(noticeData6.index) + '</i></p>';
+  }
+  output += '<p>Please delete or rename column!</p>';
+  return output;
+};
```

Comment:
Is this file automatically generated from the soy file?
If so, is there a need to commit it?

Reply:
Yes it is automatically generated. If I don't commit it, if we try run in master, the code won't work as this file is required. And to generate this file, we need to run a downloaded jar file. Should I commit the jar file and its required library with a README on how to generate the file instead?

Reply:
Can we configure gradle to generate the javascript file automatically as part of the build?

Reply:
I will have a look at this option

Reply:
Fixed.